docs/ai/tutorials/evaluate-safety.md

```diff
@@ -0,0 +1,148 @@
+
+   Response caching functionality is supported and works the same way regardless of whether the evaluators talk to an LLM or to the Azure AI Foundry Evaluation service. The response will be reused until the corresponding cache entry expires (in 14 days by default), or until any request parameter, such as the the LLM endpoint or the question being asked, is changed.
+
+1. Add a method to define the [chat options](xref:Microsoft.Extensions.AI.ChatOptions) and ask the model for a response to a given question.
+
+   :::code language="csharp" source="./snippets/evaluate-safety/MyTests.cs" id="GetResponse":::
+
+   The test in this tutorial evaluates the LLM's response to an astronomy question. Since the <xref:Microsoft.Extensions.AI.Evaluation.Reporting.ReportingConfiguration> has response caching enabled, and since the supplied <xref:Microsoft.Extensions.AI.IChatClient> is always fetched from the <xref:Microsoft.Extensions.AI.Evaluation.Reporting.ScenarioRun> created using this reporting configuration, the LLM response for the test is cached and reused.
+
+1. Add a method to validate the response.
+
+   :::code language="csharp" source="./snippets/evaluate-safety/MyTests.cs" id="Validate":::
+
+   > [!TIP]
+   > One of the checks the validation performs is that there are no diagnostics with a severity of warning or higher in each metric. Some of the evaluators, for example, <xref:Microsoft.Extensions.AI.Evaluation.Safety.ViolenceEvaluator>, produce a warning if you only evaluate the response and not the message. Similarly, they expect an equal number of messages and responses in the data you pass to <xref:Microsoft.Extensions.AI.Evaluation.Reporting.ScenarioRunExtensions.EvaluateAsync*>. However, even though an evaluator might produce a warning diagnostic in these cases, it still proceeds with the evaluation.
+
+1. Finally, add the [test method](xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute) itself.
+
+   :::code language="csharp" source="./snippets/evaluate-safety/MyTests.cs" id="TestMethod":::
+
+   This test method:
+
+   - Creates the <xref:Microsoft.Extensions.AI.Evaluation.Reporting.ScenarioRun>. The use of `await using` ensures that the `ScenarioRun` is correctly disposed and that the results of this evaluation are correctly persisted to the result store.
+   - Gets the LLM's response to a specific astronomy question. The same <xref:Microsoft.Extensions.AI.IChatClient> that will be used for evaluation is passed to the `GetAstronomyConversationAsync` method in order to get *response caching* for the primary LLM response being evaluated. (In addition, this enables response caching for the LLM turns that the evaluators use to perform their evaluations internally.)
```

Comment:
In this case, the evaluators internally talk to the Azure AI Foundry Evaluation service. So, it may be better to say something like `this enables response caching for the responses that the evaluators fetch from the Azure AI Foundry Evaluation service as part of performing their evaluations`...